app/lib/features/news/widgets/news_item_slide/video_slide.dart

```diff
@@ -37,17 +41,21 @@ class _VideoSlideState extends State<VideoSlide> {
 
   @override
   Widget build(BuildContext context) {
-    return FutureBuilder<Uint8List>(
-      future: newsVideo.then((value) => value.asTypedList()),
-      builder: (BuildContext context, AsyncSnapshot<Uint8List> snapshot) {
-        if (snapshot.hasData) {
-          return ActerVideoPlayer(
-            videoFile: File.fromRawPath(snapshot.data!),
-          );
-        } else {
-          return const Center(child: Text('Loading video'));
-        }
-      },
+    return Container(
+      color: widget.bgColor,
+      alignment: Alignment.center,
+      child: FutureBuilder<Uint8List>(
+        future: newsVideo.then((value) => value.asTypedList()),
+        builder: (BuildContext context, AsyncSnapshot<Uint8List> snapshot) {
+          if (snapshot.hasData) {
+            return ActerVideoPlayer(
+              videoFile: File.fromRawPath(snapshot.data!),
+            );
+          } else {
+            return const Center(child: Text('Loading video'));
```

Comment:
This doesn't need to happen necessarily in this PR, but a placeholder here would be nice or having ```Skeletonizer()``` here.

Reply:
yeah, loading ahead the next screen below as well ... to decrease loading time in the first place. but both for the future...